src/roswire/proxy/file.py

```diff
@@ -186,6 +186,28 @@ def islink(self, path: str) -> bool:
         code, output, duration = self.__shell.execute(cmd)
         return code == 0
 
+    def find(self,
+             path: str,
+             filename: str
+             ) -> List[str]:
+        """
+        Returns a list of files that match the provided filename
+        in a given directory recursively.
+
+        Parameters:
+            path: absolute path to the directory.
+            filename: the name of the file to match.
+
+        Raises:
+            OSError: if find operation is not successful.
+        """
+        cmd = f'find {shlex.quote(path)} -name {shlex.quote(filename)}'
+        code, output, duration = self.__shell.execute(cmd)
+        if code != 0:
+            raise OSError(f"Find operation was unsuccessful on {path}")
+        paths: List[str] = output.replace('\r', '').split('\n')
```

Comment:
Will this return the absolute file path? It might be worth writing a test for this method.

Reply:
I think it returns `path`+subdirectories. So if `path` is absolute, the returned paths are absolute too.

Reply:
I'll add a test.